static/index.html

```diff
@@ -24,10 +24,11 @@
     <button id="logout-no-clientid">logout (no client id)</button>
     <script type="text/javascript">
       $(function() {
-        createAuth0Client({
-          domain: 'auth.brucke.club',
-          client_id: 'wLSIP47wM39wKdDmOj6Zb5eSEw3JVhVp'
-        })
+        createAuth0Client
+          .default({
```

Comment:
This is going to be an issue, as this will break existing users who pull our library in through the CDN. However, I understand that for this module type there's no way we can have both named and default exports as they just overwrite each other.